PageViews/PageViewsRouter.js

```diff
@@ -0,0 +1,141 @@
+import {useRef, useCallback, Children} from 'react';
+import EnactPropTypes from '@enact/core/internal/prop-types';
+import useChainRefs from '@enact/core/useChainRefs';
+import PropTypes from 'prop-types';
+
+import {useAutoFocus} from '../internal/Panels';
+
+import useFocusOnTransition from '../WizardPanels/useFocusOnTransition';
+import useToggleRole from '../WizardPanels/useToggleRole';
```

Comment:
What about moving this hook out of WizardPanels? Its implementation is not dependent on any other component including WizardPanels.